languages/csharp/exercises/concept/properties/.docs/introduction.md

```diff
@@ -0,0 +1,19 @@
+A `property` in C# is a member of a class that provides access to attributes of that class.
+Callers can set or retrieve the attribute.  Properties can be either auto-implemented or
+have a backing field.  
+
+When setting a property the input value can be validated, formatted
+or otherwise manipulated and in fact any programmatic operation accessible to code in the
+class can be executed.  
+
+Similarly when retrieving a property data can be calculated or formatted and again
+any programmatic operation available to the class can be executed.
+
+Properties have access modifiers (`public`, `protected` etc.) in the same way as other
```

Comment:
I would suggest to only use `public` and `private`, as I envision the `protected` modifier to only be discussed in the Inheritance exercise.

```suggestion
Properties have access modifiers (`public`, `private` etc.) in the same way as other
```